api/app/tests/test_coordinates.py

```diff
@@ -0,0 +1,61 @@
+import json
+import os
+
+from django.conf import settings
+from django.contrib.gis.geos import Point
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+from rest_framework.test import APITestCase
+
+from signals.apps.signals.serializers import NearAmsterdamValidatorMixin
+
+IN_AMSTERDAM = (4.898466, 52.361585)
+OUTSIDE_AMSTERDAM = tuple(reversed(IN_AMSTERDAM))
+
+
+class TestNearAmsterdamValidatorMixin(TestCase):
+    def test_validate_geometrie(self):
+        # note this test bypasses the API
+        correct = Point(IN_AMSTERDAM)
+        wrong = Point(OUTSIDE_AMSTERDAM)
+
+        # check that valid data is returned, and wrpng data rejected
+        v = NearAmsterdamValidatorMixin()
+        self.assertEquals(correct, v.validate_geometrie(correct))
+
+        with self.assertRaises(ValidationError):
```

Comment:
Suggestion, most of the time I split this two tests scenario's into two test cases. This one is very small so it's up to you.

Reply:
Since they are small and need the same setup I would rather keep them together.